### PR TITLE
Translations: extract translation keys

### PR DESCRIPTION
**Pull request by Pineirin**
* fix translation keys.
* closes: #50

### CODE REVIEW

File: invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
     "postextract_messages": "i18next-conv -l en -s ./messages/en/translations.json -t ./translations.pot",
     "compile_catalog": "node ./scripts/compileCatalog.js",
     "init_catalog": "node ./scripts/initCatalog"
+  },
+  "dependencies": {
+    "acorn": "^8.8.0"
```
Comment (jrcastro2) [inline invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json:23]: I don't think this is not needed, might be that while testing with different dependencies you didn't delete the `package-lock.json` and it was failing. Try to delete your local node_modules (if any) and also delete the `package-lock.json` and run `npm install` it should work correctly.

P.S. While adding new dependencies keep in mind that if it's added in `dependencies` it means that it's required in production, in this case it should be in `devDependencies`.

**Comment by jrcastro2** [comment]
It would be nice if you can add to the [index](https://github.com/inveniosoftware/invenio-administration/blob/main/invenio_administration/assets/semantic-ui/translations/invenio_administration/messages/index.js) the missing `el` language.

File: invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json (hunk 1 above)
Comment (Pineirin) [reply invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json:23]: The part that puzzles me is that, while acorn was not getting installed without fixing the dependency, other acorn related packages such as acorn-class-fields and acorn-jsx where getting installed. I tried deleting the node-modules and the package-lock, but basic acorn was still not getting installed.

Comment (kpsherva) [reply invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json:23]: was that package needed? What it is needed for? 
If it is needed why it is not installed as devDependency ?

File: invenio_administration/errors.py [hunk 2]
```diff
@@ -16,15 +16,27 @@ class InvalidResource(Exception):
 
     def __init__(self, resource, view):
         """Initialise error."""
-        super().__init__(_(f"Invalid resource {resource} configured for view {view}."))
+        super().__init__(
+            _(
+                "Invalid resource {resource} configured for view {view}.".format(
```
Comment (kpsherva) [inline invenio_administration/errors.py:21]: why was this changed to use format function ?

File: invenio_administration/translations/de/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+"Language: de\n"
+"Language-Team: de <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.9.1\n"
+
+#: invenio_administration/errors.py:21
+msgid "Invalid resource {resource} configured for view {view}."
```
Comment (kpsherva) [inline invenio_administration/translations/de/LC_MESSAGES/messages.po:23]: how was this file generated?

File: invenio_administration/assets/semantic-ui/translations/invenio_administration/messages/de/translations.json [hunk 4]
```diff
@@ -3,5 +3,8 @@
   "There is no resources in this category.": "",
   "No matching resources found.": "",
   "results per page": "",
-  "result(s) found": ""
+  "result(s) found": "",
```
[the remark this answers is not in the document]
Comment (Pineirin) [reply invenio_administration/assets/semantic-ui/translations/invenio_administration/messages/de/translations.json:6]: The translation keys were extracted as explained in the doc https://inveniordm.docs.cern.ch/develop/best-practices/i18n/#python_2. The only difference is that the use-fuzzy variable was not set, as in the other repositories.

File: invenio_administration/translations/de/LC_MESSAGES/messages.po (hunk 3 above)
Comment (Pineirin) [reply invenio_administration/translations/de/LC_MESSAGES/messages.po:23]: The translation keys were extracted as explained in the doc https://inveniordm.docs.cern.ch/develop/best-practices/i18n/#python_2. The only difference is that the use-fuzzy variable was not set, as in the other repositories.

File: invenio_administration/errors.py (hunk 2 above)
Comment (Pineirin) [reply invenio_administration/errors.py:21]: The process of automatically extracting string keys was failing for this file due to using string interpolation. Using format instead it works properly.

**Review by kpsherva — APPROVED** [review]
